gradle/libs.versions.toml

```diff
@@ -37,7 +37,7 @@ spotbugs-plugin = "6.0.17"
 spotless-plugin = "6.25.0"
 
 # Packaging
-shadow-jar-plugin = "8.1.1"
+shadow-jar-plugin = "8.3.6"
```

Comment:
The orthogonal change should be handled independently because the PR title should be the main scope of the code.

Please remove this part from this PR.